Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM registry.suse.com/suse/sle15:15.3
-ENV ISTIO_VERSION 1.11.4
+ENV ISTIO_VERSION 1.10.4
```

Comment:
I don't think it's a good idea to downgrade the `ISTIO_VERSION` in master just to be able to backport a release. If we need to be able to backport changes to prior Istio release lines, we should either have multiple Dockerfiles or multiple branches like `release/v1.10` that create the releases.

Reply:
@aiyengar2 On this repository the way changes are getting added is
- Someone made some changes
- The PR gets merged to master
- Create a new tag from master (that build and push a new istio-installer image)
and same way is follows. 
We also did the same for istio 1.11.4 https://github.com/rancher/istio-installer/pull/45
For istio 1.10.4 everything remains same on master just the ISTIO_VERSION gets updated everytime and new version gets added to istio_version_array in `scripts/fetch_istio_releases.sh`

Reply:
as discussed offline, this is now fine since we are initializing a 1.10 release branch. Resolving this conversation